corehq/mobile_ucr_v2_update_script.py

```diff
@@ -37,7 +37,7 @@
 skip_domains = set()
 
 
-def process():
+def process(dry_run=False):
```

Comment:
nit: totally up to you and fine to leave it as it is for this one, I usually keep dry run as True by default, it avoids accidental updates for me, but it is also fine either way.

Reply:
That sounds like a good idea. Addressed in [25a0298](https://github.com/dimagi/commcare-hq/pull/35158/commits/25a0298903c43d9c7dc0f9e8fe6d95ab7e4e678c).